debian_package/sbin/folder2ram

```diff
@@ -234,67 +235,40 @@ exit
 EOF
 
   #chmodding this script to make it only root/group-writable and root/group-executable
-  chmod 774 "/etc/init.d/folder2ram_temporary"
+  chmod 755 "/etc/init.d/folder2ram_temporary"
 
 } #### END write_cleanup_initscript
 
 write_systemd_startup_service() {
 
-  if [ "$timeout_setting" != '' ]; then
-    timeout_line="TimeoutSec=$timeout_setting"
-  fi
+  #read timeout from configuration file
+  local timeOUT=$(read_config_file | grep "^TIMEOUT" | awk -F'=' '{print $2}')
+  [ "${timeOUT}" != "" ] || timeOUT=120
 
   #writing a tiny amount of stuff to have systemd do the same thing as above.
   #clear symptom that systemd is master race.
   cat <<EOF >"$systemd_startup_service_file"
 [Unit]
 Description=folder2ram systemd service
 After=local-fs.target
-After=blk-availability.service
-DefaultDependencies=no
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=/sbin/folder2ram -mountall
-$timeout_line
+ExecStart=$(which journalctl) --relinquish-var
+ExecStart=-${binFILE} -mountall
+ExecStart=$(which journalctl) --flush
+ExecStop=$(which journalctl) --relinquish-var
+ExecStop=-${binFILE} -umountall
+ExecStop=$(which journalctl) --flush
+TimeoutSec=${timeOUT}
```

Comment:
I assume this is for the "modify ExecStart to suppress problems with journald" 
What is the journald issue that needs this workaround?